### PR TITLE
fix: better handle absolute paths

### PR DESCRIPTION
**Pull request by jeffin143**
**What kind of change does this PR introduce?**
fix : #10247 

**Did you add tests for your changes?**
NA

**Does this PR introduce a breaking change?**
No

**What needs to be documented once your changes are merged?**
May be that webpack-4 can now handle Absolute network paths, Not sure

### CODE REVIEW

File: schemas/ajv.absolutePath.js [hunk 1]
```diff
@@ -40,8 +38,12 @@ module.exports = ajv =>
 					];
 					passes = false;
 				}
-
-				if (!isCorrectAbsoluteOrRelativePath) {
+				// ?:[A-Za-z]:\\ - Windows absolute path
+				// \\\\ - Windows network absolute path
+				// \/ - Unix-like OS absolute path
+				const isCorrectAbsolutePath =
+					expected === /^(?:[A-Za-z]:(\\|\/)|\\\\|\/)/.test(data);
```
Comment (sokra) [inline schemas/ajv.absolutePath.js]:
```suggestion
					expected === /^(?:[A-Za-z]:\\|\\\\|\/)/.test(data);
```

correct windows paths must use `\` and not `/`.

Comment (jeffin143) [reply schemas/ajv.absolutePath.js]: Made changes

Comment (evilebottnawi) [reply schemas/ajv.absolutePath.js]: @sokra Windows supports `c:/foo/bar` https://nodejs.org/api/path.html#path_path_isabsolute_path

**Comment by sokra** [comment]
Thasnk

**Pull request merged**